### PR TITLE
Grb/inline text class

### PR DESCRIPTION
**Pull request by yoyis3000**
Renamed the classes of InlineText to match the other Inline element's naming convention.

### CODE REVIEW

**Comment by chrisarevir** [comment]
Inner/Outer container sound like they are the wrapping classes for the entire thing. Would it be possible to name them a little clearer?

**Comment by yoyis3000** [comment]
@chrisarevir I can totally do that, the only reason I chose those class names because the Inline elements share those class names and because of that we only need one override.scss file.. to override them all. If I rename them I'd imagine you'd want me to rename them all? Which I don't mind doing, I just want to make sure that's the route we want to go.

File: table/README.md [hunk 1]
```diff
@@ -4,6 +4,9 @@ A table component with self-managed checkboxes, sticky head, sticky right panel,
 
 # Change Log
 
+##### 4.4.0
```
Comment (ben-burlingham) [inline table/README.md]: nb `minor` semver is for new functionality; this is a breaking change so I'd probably recommend a `major` bump.

(`patch` is for nonbreaking changes that aren't new functionality)

Comment (yoyis3000) [reply table/README.md]: Ah, I wanted to make sure it was a major patch, and it was but I forgot to apply to the version numbers.

**Comment by nickwmurphy** [comment]
This is going to clash with #300. Let's chat about this.

**Pull request closed without merging**